### ISSUE TITLE
BO - PHP Deprecated hook warning displayed in statsdata module

### ISSUE THREAD

**Issue by nl2ttl** [PrestaShop/PrestaShop#31529]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

PHP errorlog shows:

PHP Deprecated:  The hook "createAccount" is deprecated, please use "actionCustomerAccountAdd" instead in module "statsdata". in /public_html/classes/Hook.php on line 850

### Expected behavior

No error :)

### Steps to reproduce

1. Install 8.0.1, update statsdata to latest version.
2. Register a customer in FO.
3. Check error logs.

### PrestaShop version(s) where the bug happened

8.0.1

### PHP version(s) where the bug happened

7.4.33

**Comment by Hlavtox** [comment]
@nl2ttl Good catch, do you think you could open a PR in the module repository changing this hook and adding an upgrade process [like here](https://github.com/PrestaShop/statsdata/blob/dev/upgrade/upgrade-2.1.1.php)? There was already one hook fixed like this for the current version.

**Comment by nl2ttl** [comment]
@Hlavtox sorry I don't understand what you want me to do.

**Comment by Hlavtox** [comment]
@nl2ttl I was asking, if you you wanted to contribute to this project and try to fix the issue in the module, so everyone can benefit from it. 👍

**Comment by PrestaEdit** [comment]
Linked PR : https://github.com/PrestaShop/statsdata/pull/32

**Comment by florine2623** [comment]
Issue is fixed and will be available in statsdata v2.1.2.
